### PR TITLE
Re-did the navigation notification layout, and added a separate layou…

### PR DESCRIPTION
**Pull request by devotaaabel**
…t for when it is expanded vs. collapsed. I have updated the images to reflect a notification that is consistent in size and style across all devices and versions of android. I'll create another PR with the `TimeUtils` updates.

cc. @Cjballard @danesfeder @Guardiola31337 

device-2018-06-27-171043
device-2018-06-27-171021

### CODE REVIEW

**Comment by danesfeder** [comment]
cc @Cjballard for eyes on the layout updates

**Comment by Cjballard** [comment]
Looks good – I might make one slight change and line up the left edge of the "X End Navigation" action with the instruction text above, like attached. 

banner

**Comment by devotaaabel** [comment]
Better @Cjballard ?

device-2018-06-25-141533

**Comment by Cjballard** [comment]
@devotaaabel Looks good!

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java [hunk 1]
```diff
@@ -182,6 +187,8 @@ private void updateArrivalTime(RouteProgress routeProgress) {
       currentArrivalTime = TimeUtils.formatArrivalTime(routeProgress.durationRemaining());
       notificationRemoteViews.setTextViewText(R.id.notificationArrivalText,
         String.format(Locale.getDefault(), "%s ETA", currentArrivalTime));
+      bigNotificationRemoteViews.setTextViewText(R.id.notificationArrivalText,
+        String.format(Locale.getDefault(), "%s ETA", currentArrivalTime));
```
Comment (Guardiola31337) [inline libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java]: We should extract `ETA` from here and internationalize it (i.e. add it to the translations).

Comment (Guardiola31337) [reply libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java]: Why is not `ETA` appearing at all (not shown in the above screenshots)? 🤔

**Comment by akitchen** [comment]
Per planning meeting, this is related to https://github.com/mapbox/mapbox-navigation-android/issues/713

**Comment by Guardiola31337** [comment]
Noticed that we're not showing the street in devices with low resolution/screen density 👀

motox_collapsed_notification_issue

_Moto X 2nd gen_ ☝️

nexus5_collapsed_notification

_Nexus 5_ ☝️ 

cc @devotaaabel

File: libandroid-navigation/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -7,4 +7,5 @@
     <string name="meters">m</string>
     <string name="miles">mi</string>
     <string name="feet">ft</string>
+    <string name="eta_format">\"%s ETA\"</string>
```
[the remark this answers is not in the document]
Comment (devotaaabel) [reply libandroid-navigation/src/main/res/values/strings.xml]: Oops, I didn't mean for those to be escaped. And not sure about Transifex, cc: @danesfeder?

**Comment by devotaaabel** [comment]
@Guardiola31337 updated

**Pull request merged**